Create jcenter.json

A-AFTAHI opened the pull request:
This file contain 251 new libraries metadata from Jcenter Repo to be used with the libraries scraper

A-AFTAHI committed:
Create jcenter.json

This file contain 251 new libraries metadata from Jcenter Repo to be used with the libraries scraper

reddr commented:
I just tried to download the list. Apparently, you dumped all entries from the jcenter index.
There are many items that do not have a single valid version or are missing a maven-metadata.xml that contains the version history.

Thus, please remove any item, that does not have a single valid version. For this you can check the script output (no version and URL/HTTP error).

A-AFTAHI committed:
update jcenter.json

183 libraries metadata to use with scraper.py after deleting all previous libraries with http/url errors or the ones with no versions available.

A-AFTAHI commented:
please check the changes made to the file content.
